content/en/docs/tasks/access-kubernetes-api/custom-resource-definition-versioning.md

```diff
@@ -0,0 +1,238 @@
+{{% capture steps %}}
+
+## Overview
+
+Kubernetes standard API types include a versioning mechanism, so that developers
+can evolve APIs while maintaining backwards compatibility. In a similar way, the
+CustomResourceDefinition API supports a `versions` field that you can use to
+support multiple versions of Custom Resources that you have developed.
+
+{{< note >}}
+Earlier iterations included a `version` field instead of `versions`. The
+`version` field is deprecated and optional, but if it is not empty, it must
+match the first item in the `versions` field.
+{{< /note >}}
+
+## Specify multiple versions
+
+This example shows a CustomResourceDefinition with two versions. The comments in
+the YAML provide more context.
+
+```yaml
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  # name must match the spec fields below, and be in the form: <plural>.<group>
+  name: crontabs.example.com
+spec:
+  # group name to use for REST API: /apis/<group>/<version>
+  group: example.com
+  # list of versions supported by this CustomResourceDefinition
+  versions:
+  - Name: v1beta1
+    # Each version can be enabled/disabled by Served flag.
+    Served: true
+    # One and only one version must be marked as the storage version.
+    storage: true
+  - Name: v1
+    Served: true
+    storage: false
+  # either Namespaced or Cluster
+  scope: Namespaced
+  names:
+    # plural name to be used in the URL: /apis/<group>/<version>/<plural>
+    plural: crontabs
+    # singular name to be used as an alias on the CLI and for display
+    singular: crontab
+    # kind is normally the CamelCased singular type. Your resource manifests use this.
+    kind: CronTab
+    # shortNames allow shorter string to match your resource on the CLI
+    shortNames:
+    - ct
+```
+
+You can save the CustomResourceDefinition in a YAML file, then use
+`kubectl create` to create it.
+
+```shell
+kubectl create -f my-versioned-crontab.yaml
+```
+
+After creation, the API server starts to serve each enabled version at an HTTP
+REST endpoint. In the above example, the APIs are available at
+`/apis/example.com/v1beta1` and `/apis/example.com/v1`.
+
+### Version order
+
+The versions in a CustomResourceDefinition are automatically prioritized in the
+following way, regardless of the order in which they are defined. The version
+that comes first in term of priority is used by kubectl as the default version
+to access objects. 
+
+The version _name_ in the CustomResourceDefiniton version list is used to
+compute the order of versions, using the following algorithm to attempt to parse
+the versions by version, then stability level. 
+
+Versions of the Kubernetes versioning scheme start with a `v` followed by a
+number, `vbeta` followed by a number, or `vbeta` followed by a number. The
+Kubernetes community calls this scheme "GA versions". Within GA versions, the
+following rules apply:
+  
+- `v` is sorted before `vbeta`, which is sorted before `valpha`.
```

Comment:
still confusing to me to use vbeta. Not sure if vNbeta is better but it suggest there is a number in between.